src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -61,6 +61,8 @@ public abstract class SqlMetadataProvider<ConnectionT, DataAdapterT, CommandT> :
 
         private Dictionary<string, string> EntityPathToEntityName { get; } = new();
 
+        private Dictionary<string, string> EntityNameToEntityPath { get; } = new();
```

Comment:
If this is no longer populated, do we still need the member?

Reply:
Oh no, we don't. Must've missed removing this. Thanks!